src/components/GamePage.tsx

```
@@ -53,21 +84,21 @@ const Game: React.FC = () => {
     try {
       const move = makeMove(sourceSquare, targetSquare, 'H');
       if (move === null) {
-        alert('Invalid move. Please try again.');
+        showAlert('Invalid move. Please try again.', 'warning');
         return false;
       }
       setTimeout(handleAIMove, 500);
       return true;
-    } catch {
-      setError('An error occurred while making your move.');
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)